build-system/tasks/helpers.js

```diff
@@ -555,8 +557,6 @@ async function minify(code, map) {
     nameCache,
   };
   // TS complains if defined inline, since it sees type `string` but needs type ("strict" | boolean).
```

Comment:
Missed this.

Reply:
was able to fix with a silly type case 🤷